### PR TITLE
Add guide for using webhook step in automations

### PR DESCRIPTION
**Pull request by richard-rance**
Add a guide for using the webhook action of the automations system.

[Preview Link](https://developers.vendasta.com/platform/branches/webhook-docs/ZG9jOjU2MTQxMDE2-use-the-trigger-a-webhook-action)

@vendasta/autobots @vendasta/external-apis 

EA-424

### CODE REVIEW

**Review by bbudz-va — APPROVED** [review]
wondering if it would be valuable to have a link to a resource article for this action (@clim-va) and/or vice versa so when we make a new release they're tied together to update

File: docs/Guides/Automate/WebhookAction.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+# Use the trigger a webhook action
+
+You can take action in your systems based on events in the Vendasta platform by using automations. With any automation you can add the `Trigger a webhook` step to be notified when something happens. 
+
+## Overview
+
+In this guide we will create an automation, listen for the event and retrieve details of the affected records.
+
+## Pre requsits
```
Comment (coreyhickson) [inline docs/Guides/Automate/WebhookAction.md:9]:
```suggestion
## Prerequisites
```

**Review by coreyhickson — APPROVED** [review]

**Comment by telmoselhi-va** [comment]
> wondering if it would be valuable to have a link to a resource article for this action (@clim-va) and/or vice versa so when we make a new release they're tied together to update

@bbudz-va I'll be linking the updated docs to Erica so she can update the resource article

File: docs/Guides/Automate/WebhookAction.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+## Step 1: Automation setup
+
+In Partner Center go to the automations screen and create an automation based on the trigger and criteria that you are interested in.
+
+Add the `Trigger a webhook` step.
+
+![](webhookAddPart1.png)
+
+The **webhook URL** will be an endpoint that you have built in step 2. It will likely be one of your web servers but could also be a 3rd party system like Zapier.
```
Comment (echessall) [inline docs/Guides/Automate/WebhookAction.md:22]: step 2 [of the automation] maybe?

File: docs/Guides/Automate/WebhookAction.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+In this guide we will create an automation, listen for the event and retrieve details of the affected records.
+
+## Prerequisites
+
+This guide assumes you are familiar with creating automations within the platform. If you are not familiar you should [learn more about automations](https://support.vendasta.com/hc/en-us/sections/4406950706583-Automations) first.
+
+
+## Step 1: Automation setup
+
+In Partner Center go to the automations screen and create an automation based on the trigger and criteria that you are interested in.
```
[the remark this answers is not in the document]
Comment (telmoselhi-va) [reply docs/Guides/Automate/WebhookAction.md:16]: how do you feel about arrows over `>`? 
image

File: docs/Guides/Automate/WebhookAction.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+Add the `Trigger a webhook` step.
+
+![](webhookAddPart1.png)
+
+The **webhook URL** will be an endpoint that you have built in step 2. It will likely be one of your web servers but could also be a 3rd party system like Zapier. 
+
+For testing purposes [webhook.site](https://webhook.site/) is a great tool for viewing what is sent. You may also use a tool like [ngrok](https://ngrok.com/) to route requests to your local development computer.
+
+The **Verifier Token** is something that you can makeup. We will send it along with the webhook to help you verify that the request actually came from us.
```
Comment (echessall) [inline docs/Guides/Automate/WebhookAction.md:26]: Maybe a bit more explanation here. Maybe something like The Verifier Token is named by you and is sent along with the webhook to help you verify that the request actually came from us.

File: docs/Guides/Automate/WebhookAction.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+Custom **Additional Fields** may be added to the request body. Be careful when including sensitive data. The requests are sent using https however we do not have a way to confirm the destination is under your team's control. Instead it is recommended to only send ids and then fetch up to date info using an API request.
```
Comment (echessall) [inline docs/Guides/Automate/WebhookAction.md:28]: ...confirm the destination **and** this is under your team's control.

File: docs/Guides/Automate/WebhookAction.md (hunk 4 above)
Comment (telmoselhi-va) [reply docs/Guides/Automate/WebhookAction.md:26]: how about
The Verifier Token is designated by you and is sent along with the webhook to help you verify that the request originated from this automation.

File: docs/Guides/Automate/WebhookAction.md (hunk 2 above)
Comment (telmoselhi-va) [reply docs/Guides/Automate/WebhookAction.md:22]: I think `step 2` in this case refers to step 2 of this guide

Comment (echessall) [reply docs/Guides/Automate/WebhookAction.md:22]: Oh I see!! Yes maybe do that instead!

File: docs/Guides/Automate/WebhookAction.md (hunk 4 above)
Comment (echessall) [reply docs/Guides/Automate/WebhookAction.md:26]: That's perfect!

File: docs/Guides/Automate/WebhookAction.md (hunk 5 above)
Comment (echessall) [reply docs/Guides/Automate/WebhookAction.md:28]: Oh sorry I read that wrong. Now I understand what you're saying!

**Pull request merged**